src/core/manager.c

```diff
@@ -2931,11 +2931,9 @@ int manager_loop(Manager *m) {
                 if (manager_dispatch_dbus_queue(m) > 0)
                         continue;
 
-                /* Sleep for half the watchdog time */
+                /* Sleep for watchdog runtime wait time */
                 if (timestamp_is_set(m->runtime_watchdog) && MANAGER_IS_SYSTEM(m)) {
-                        wait_usec = m->runtime_watchdog / 2;
-                        if (wait_usec <= 0)
-                                wait_usec = 1;
+                        wait_usec = watchdog_runtime_wait();
```

Comment:
per our coding style: please remove the {} around single line if blocks, see https://systemd.io/CODING_STYLE

Reply:
let's also move the `timestamp_is_set()` part of the if check into watchdog_runtime_wait() please. If we introduce an encapsulated function for this we might as well encapsulate properly.